cogip/entities/dynobstacle.py

```diff
@@ -53,3 +53,46 @@ def set_size(self, length: int, width: int) -> None:
         """
         self.mesh.setXExtent(width)
         self.mesh.setYExtent(length)
+
+
+class DynRoundObstacleEntity(Qt3DCore.QEntity):
```

Comment:
Maybe could we keep names aligned with mcu-firmware and replacing 'Round' by 'Circle' ?

Reply:
Nevermind, I saw your further commit.